native/packages/react-native-bpk-component-navigation-bar/readme.md

```diff
@@ -0,0 +1,185 @@
+# react-native-bpk-component-navigation-bar
+
+> Backpack React Native navigation bar component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-navigation-bar --save-dev
+```
+
+## Usage
+
+The component has slightly different APIs on iOS and Android. For both platforms the `BpkNavigationBar` is the main component
+to use. On Android we additionally provide `BpkNavigationBarButtonAndroid` to render the leading and trailing bar buttons. On iOS
+there are a few more variations of buttons and we provide three components:
+
++ `BpkNavigationBarBackButtonIOS` for rendering a back item with an optional title.
++ `BpkNavigationBarTextButtonIOS` for rendering a text button.
++ `BpkNavigationBarIconButtonIOS` for rendering an icon only button.
+
+### Android
+
+```js
+import React from "react";
+import { I18nManager } from "react-native";
+import BpkNavigationBar, {
+  BpkNavigationBarButtonAndroid
+} from "react-native-bpk-component-navigation-bar";
+
+const backIcon = () =>
+  I18nManager.isRTL ? "native-android--forward" : "native-android--back";
+
+export default () => (
+  <BpkNavigationBar
+    leadingButton={
+      <BpkNavigationBarButtonAndroid
+        title={translationHelper.t("back-button-title")}
```

Comment:
We are not using `translationHelper` consistently across readme's so lets remove them.

Reply:
This seems like the wrong conclusion